demo/api/src/news/blocks/news-link-block-transformer.service.ts

```diff
@@ -0,0 +1,39 @@
+import { BlockTransformerServiceInterface } from "@comet/blocks-api";
+import { InjectRepository } from "@mikro-orm/nestjs";
+import { EntityRepository } from "@mikro-orm/postgresql";
+import { Injectable } from "@nestjs/common";
+
+import { News } from "../entities/news.entity";
+import { NewsLinkBlockData } from "./news-link.block";
+
+type TransformResponse = {
+    news?: {
+        id: string;
+        slug: string;
+        scope: {
+            domain: string;
+            language: string;
+        };
```

Comment:
it is not so great that the api has to do that.

Reply:
I agree. It's also not so nice that we already do this for the internal link block.

I can think of two alternatives:
- Use a context to provided the base path info to all child components (we already discussed that)
- Neglect linking between different scopes and get the info using `usePathname()`:

  ```ts
  const pathname = usePathname();
  const language = pathname.slice(3); // "/en"
  ```
  Downside: Only works on the client, could lead to hydration errors

I'm not sure if we ever link from one scope to another.

Reply:
Discussed this in last week's meeting and decided that transformToPlain is okay. We need to find a solution for the block meta generation of nested objects, but that's out of scope of this PR.